modules/ROOT/partials/integrations/angular-tech-ref.adoc

```diff
@@ -258,13 +287,17 @@ Used to specify the format of the content emitted by the `tinymce-angular` compo
 
 ===== Example: Using `outputFormat`
 
-```html+++<editor outputFormat="text">++++++</editor>+++
-
-```
+[source, html]
+----
+<editor
+  outputFormat="text">
+</editor>
+----
 
+[[tagname]]
 ==== `tagName`
 
-Only valid when <<inline,`<editor [inline]="true"></editor>`>>. Used to define the HTML element for the editor in inline mode.
+Only valid when pass:[`<editor [inline]="true"></editor>`]. Used to define the HTML element for the editor in inline mode.
```

Comment:
```suggestion
Only valid when xref:inline[`+<editor [inline]="true"></editor>+`]. Used to define the HTML element for the editor in inline mode.
```
This conversion isn't correct as you've used lost the `xref`. Also the `pass` macro generally shouldn't be used as it means this would be interpreted as HTML if you did that (which was the case here and it was rendering badly).

Reply:
sorry, I don't know how I came to this solution and missing that I did not put the `xref` in it, I had change it to your solution